insonmnia/worker/server.go

```diff
@@ -1096,6 +1096,70 @@ func (m *Worker) setupRunningContainers() error {
 	return nil
 }
 
+func (m *Worker) tasksGC() error {
```

Comment:
I think maybe it's better to do it only once on start

Reply:
we are trying to fix a state that results from race condition (that does not necessarily occur only when someone shuts the worker down)